src/main/flight/servos.c

```diff
@@ -383,6 +397,14 @@ void writeServos(void)
         servoIndex += 2;
     }
 
+    // Scan servos and write those marked forwarded and not written yet
+    for (int i = 0; i < MAX_SUPPORTED_SERVOS; i++) {
```

Comment:
Interesting. This functionality seems to be competing with the one in `forwardAuxChannelsToServos()`, and I don't think it is in a good way.
This really needs a clean up.

Reply:
I think it is okay to let `forwardAuxChannelsToServos()` output whatever AUX channel values are to remaining servos. The purpose of this loop is to handle servos that are configured but not part of the `smix` and gimbal. (And servo index is still kept tracked by `servoIndex++`.)

Servo write orders
1. `smix` defined servos
2.  gimbal servos
3. `servo` defined servos that are not `smix` defined servos
4. `FEATURE_CHANNEL_FORWARDING` servos

Reply:
I see. I wasn't aware that this is meant to be supported, as to me the documentation (https://github.com/betaflight/betaflight/blob/master/docs/Mixer.md#cli-servo) reads as if `servo` can only be used in conjunction with `smix`.

This documentation has always been in dire need of an update, but I am not enough of a servo user to be able to tell what use cases the functionality is actually trying to cover.